org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolutionAcceptor.java

```diff
@@ -38,6 +38,17 @@ public void accept(String label, IModification<EObject> modification) {
 		issues.add(new DiagnosticResolution(label, modificationContextFactory, modification));
 	}
 
+	public List<DiagnosticResolutionInfo> getDiagnosticResolutionInfos() {
+		List<DiagnosticResolutionInfo> infos = new ArrayList<>();
+		int id = 0;
+		for (DiagnosticResolution issue : issues) {
```

Comment:
Can we come up with a better name for the field `issues` and the local variable `issue`?

Reply:
issueResolution : resourceIssueResolutions